src/extension/extension.ts

```diff
@@ -203,8 +197,6 @@ export class RunmeExtension {
 
     await resetNotebookSettings()
 
-    const transientOutputs = !getSessionOutputs()
```

Comment:
It seems that for the current usage of cache + in memory session outputs, it should be always `transientOutputs: false.`